tests/support/client.rs

```diff
@@ -4,14 +4,10 @@ use std::io;
 use std::sync::Mutex;
 
 use self::futures::sync::{mpsc, oneshot};
-use self::tokio::{
-    io::{AsyncRead, AsyncWrite},
-    net::TcpStream,
-};
-use self::tokio_rustls::TlsStream;
+use self::tokio::net::TcpStream;
 use self::webpki::{DNSName, DNSNameRef};
+use self::RunningIo;
```

Comment:
I don't think this is necessary -- was it a find-and-replace?

Reply:
most likely